skyscrapers/charts
# Fix elasticsearch-monitoring alerts

- Bump to include latest upstream chart versions
- Remove ServiceMonitors, they're included upstream
- Use new labeling conventions
- Add helper functions to truncate exporter job names
- Make sure prometheus job selectors use truncated names, fixing the broken alerts on some deployments

As per https://github.com/skyscrapers/engineering/issues/304
